images/Dockerfile.gpu.allensdk

```diff
@@ -0,0 +1,73 @@
+FROM quay.io/jupyter/datascience-notebook:python-3.11.10
```

Comment:
Would it make sense to use

`FROM dandiarchive/dandihub:latest-allensdk`

so we can reduce the duplication?

Reply:
We could also build many of these from dandihub:latest, but that would require a bit of a refactor, specifically, we'd want to make sure that latest is built (and pushed if being done in separate environments) prior to the others building.  IMO this could be regarded as a future improvement, not blocking merge.

Reply:
I think that the only issue will be that we tag everything as `latest` so if the jobs that build images start simultaneously then the GPU image may pull the older `latest-allensdk` image.

Reply:
Looks like had the same idea and hit `Enter` at the same time.  That sounds good.  We can implement this in the future.